recipes/mothur/meta.yaml

```diff
@@ -1,35 +1,48 @@
-{% set version = "1.42.1" %}
+{% set version = "1.42.3" %}
+{% set sha256 = "c723c5204ccefe4b598b9d7dee12a8193154849808e77d896ce3ca5265e9c352" %}
 
 package:
   name: mothur
   version: {{ version }}
 
 source:
-  url: https://github.com/mothur/mothur/archive/v{{ version }}.tar.gz
-  sha256: 6b61591dda289ac2d8361f9c1547ffbeeba3b9fbdff877dd286bad850bbd5539
+  - url: https://github.com/mothur/mothur/archive/v.{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+  # - url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.5/src/hdf5-1.10.5.tar.gz
```

Comment:
can you clean up this recipe a little bit?

Reply:
Already planned on it. I was having issues with linking/libraries so those were remnants from my backup plan.